src/components/Rockets/rockets.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Badge, Card, Button,
+} from 'react-bootstrap';
+import { reserveRocket, cancelRocketReservation } from '../../redux/Rockets/rockets';
+
+const Rocket = ({ rocket }) => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.Rockets);
+  const {
+    id, rocketName, description, rocketImage, reserved,
+  } = rocket;
+  const handleReserve = (e) => {
+    if (!reserved) {
+      dispatch(reserveRocket(+e.target.id));
+      console.log('reserved...', rockets);
```

Comment:
- [ ] Please do not commit console.log to your repo. It really kills performance and can make confidential data to be visible to anyone using the browser tools to look at your website.

![Screenshot from 2022-09-29 00-01-23](https://user-images.githubusercontent.com/63087491/192903415-2d38a78c-1aba-4166-9b6c-1182326c1ae4.png)

It's ugly.

kindly remove all console.log lines in your codebase.